locales/en.yml

```diff
@@ -97,6 +97,10 @@ en:
             }
         invalid_plugin_settings: >-
           Something is wrong with your configuration.
+        invalid_plugin_settings_duplicated_keys: |-
+          Duplicated keys found in config file: [%{keys}]
```

Comment:
I think we should say "Duplicate keys found" ... ?
Also "in your configuration" instead of "config file"  since configs are not always files